src/NHibernate.Test/NHSpecificTest/NH3386/Fixture.cs

```diff
@@ -60,7 +60,7 @@ public void ShouldSupportNonRuntimeExtensionWithoutEntityReference()
 
 	public static class SqlServerFunction
 	{
-		[LinqExtensionMethod]
+		[LinqExtensionMethod(DBOnly = true)]
```

Comment:
I am going to cherry-pick that newid test into the NH4 branch I have just made to showcase this.